modules/server/src/topics/getpropertytopic.cpp

```diff
@@ -70,6 +72,11 @@ void GetPropertyTopic::handleJson(const nlohmann::json& json) {
     else if (requestedKey == RootPropertyOwner) {
         response = wrappedPayload(global::rootPropertyOwner);
     }
+    else if (requestedKey == SessionRecordingPlaybackList) {
```

Comment:
I think we should strive to minimize the number of these special case keys in `getpropertytopic`. Since this one has nothing to do with the property tree, how about putting this into the session recording topic for now?
In the future we can replace this with a javascript call into the lua api (luascripttopic) and get return values from there.

Reply:
I remember not wanting to put it here vs. session recording topic, but it seemed that I had to do it this way in order for it to work in the web GUI.
In any case, I don't want to risk breaking compatibility with the GUI right now, since the session recording part of it will be changed by @emiax  and/or @micahnyc.